doc/final_report.md

```diff
@@ -0,0 +1,90 @@
+---
+
+
+```{r setup, inlude=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(knitr)
+library(kableExtra)
+library(tidyverse)
+```  
+
+# Summary 
+
+For this project we have carried out a hypothesis test to determine if there is a significant difference in the median sentence lengths between the indigenous and non-indigenous inmates under the Correction Services Canada. The median was selected as the measure of central tendency as the distribution of sentence length for both groups were skewed. Subsequently, a permutation test under the null model was carried out computationally with a significance level of 0.05. The null hypothesis states that there was no difference in the population medians in sentence length between indigenous and non-indigenous inmates. The alternate hypothesis states that there is a difference in the population medians in sentence length between indigenous and non-indigenous inmates. The resulting sample difference in the two medians was `r TEST_STATISTIC`, with a corresponding p-value of `r P_VALUE `. As this p-vaule was greater than the significance level, there was insufficient evidence to reject the null hypothesis. Therefore this study concludes that there is no statistically significant difference in the median jail times between the two groups. Considering the large sample size of `r SAMPLE_SIZE`, there is a small chance that the result is affected by Type 2 error. However, we have to be mindful of the very small effect size. 
+
+# Introduction 
+
+The number of indigenous incarcerations in Canada has increased by 40% in the prison system over the last decade, a faster rate than any other group [@ owusu2014years].  Issues such as increased domestic violence, disruptions in societal structure and norms and health issues are aggravated in highly incarcerated groups [@ freudenberg2001jails]. Hence, these rates are likely to worsen.  
+
+As this population is so vulnerable, the other key factor that should be investigated is if there is a difference in the length of the sentence between the indigenous and non-indigenous groups. This is an important question to infer as it can lead to further studies on the type of charges either group is subject too and whether particular laws disadvantage the indigenous group if a significant difference is found. Additionally, the study could also lead to an exploration of biases present in the current legal system when sentencing. 
+
+# Methods 
+
+## Data 
+
+The data set used for this study is the Offender Profile from 2017-2018 released by the Correctional Service of Canada. The link to this site can be found [here](https://open.canada.ca/data/en/dataset/844ff1e3-e137-41be-9ebe-6bd9843c1a53). Each entry in the dataset corresponds to a single offender serving a two or more year long sentence. The demographic details such as age, gender and marital status at year end are provided for each entry. This was retrieved from the Offender Management System (OMS).
```

Comment:
The data set used for this study is the Offender Profile from 2017-2018 released by the Correctional Service of Canada. The link to the online resource page can be found [here](https://open.canada.ca/data/en/dataset/844ff1e3-e137-41be-9ebe-6bd9843c1a53). Each entry in the data set corresponds to a single offender serving a two or more year long sentence. The demographic details such as age, gender and marital status at year end are provided for each entry. This was retrieved from the Offender Management System (OMS.)